package.json

```diff
@@ -62,5 +62,6 @@
     "./packages/**/*.ts": [
       "yarn lint"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
can you remove this field?

Reply:
I had reverted this but it must've slipped in again. done